### ISSUE TITLE
Blank art

### ISSUE THREAD

**Issue by RapthorneLightweaver** [malulleybovo/SymbolArtEditorOnline#1]
Upon saving the symbol art in SAR format, and saving in to the correct directory, trying to import in to the game just results in a blank symbol art

**Comment by goremageddon** [comment]
Same.

**Comment by malulleybovo** [comment]
I will need more information to identify the cause of this issue as my attempts to reproduce the issue have failed. And by "failed" I mean I do get the expected result (the correct image in game).
1) If you open this SA Editor (on Google Chrome, for instance), click on the upload icon, and select the .sar file that is not importing correctly, does this editor also show it blank? Or does it load it correctly?
2) Which browser did you use to make the corrupted .sar file? If possible, what browser version?
3) Which game server are you importing it to? Japan or North America? This editor was made entirely based on the JP server so there might be differences in the file format that are unknown to me.
4) In game, after opening the SA screen, clicking the import tab and then the import button within the import tab, do you see the SA you saved in the import folder? (even if blank, is it there?)
5) Could you include an example of a .sar file with this issue in here (by dragging and dropping the file onto the comment panel)?
Lastly, note that I maintain this repository as a hobby only so any effort put in it is out of my own free time and will. So I cannot promise any deadlines for when this will be fixed. I hope everyone can understand.

**Comment by shirikoo** [comment]
Its the overlay image, its not saving. I'm having the same issue, if I try just saving an overlay image as symbol art, it just comes out a transparent image, no matter how much or how little transparency I add it as

**Comment by malulleybovo** [comment]
I see, I think I understand the issue now so let me clarify things for you. The overlay image feature in the editor serves to guide you in creating your symbol art, it does not in fact make the art for you. Symbol Arts are made of symbols (similar idea to SVG image files) while the overlay image you import is made of pixels (JPEG, PNG image files, etc.). It may sound a little frustrating (I was myself before putting up with it and making this application), but they are two conceptually different image types, and while it is easy to convert a symbol-based image to a pixel based image, it is virtually impossible to convert it the other way around given the limit of 225 symbols that SEGA imposes on the SAR image format. So in the end (unfortunately) you have to place the symbols yourself to look as close as the original image as possible.

TL;DR: The overlay image does not add an image to the Symbol Art, it only serves as guidance for you to know where to place the symbols in order to make the Symbol Art look like the image you want.

**Comment by malulleybovo** [comment]
Closing due to inactivity. Your troubles have been heard. Although I cannot promise a way to accurately create an SA (pseudo-vector based image) from a JPG/PNG (pixel based image) using only 225 layers of symbols (SA limitation), I will make this fact clearer to the user in the next release to avoid misinterpretation.
